src/Campaigns/Blocks/CampaignList/app.tsx

```diff
@@ -0,0 +1,17 @@
+import {render} from '@wordpress/element';
+import {CampaignListType} from './types';
+import App from './app/index';
+
+/**
+ * @unreleased
+ */
+const nodeList = document.querySelectorAll('.give-campaigns-campaignListBlock-container');
```

Comment:
I feel like targeting classnames for the react app can be a little risky as there's always a chance for those to be changed. I would suggest using a data attribute instead , maybe something like `data-givewp-campaign-grid`

Reply:
Hmm, data attributes might work better than classes in the long run. Good suggestion.

Reply:
Resolved https://github.com/impress-org/givewp/pull/7708/commits/cc8f5ab9b4ef45aca1d6bef37ac0c8f643fca6fa